spec/aruba/environment_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'aruba/environment'
+require 'spec_helper'
+
+describe Aruba::Environment do
+  let(:environ) { Aruba::Environment.new  }
+
+  it 'should find PATH' do
+    expect(environ['PATH']).not_to be_nil
+  end
+
+  it '#clear' do
```

Comment:
it -> describe. Why is used `puts` here? 

`it { expect(environ.clear.to_h).to be_empty }`